hack/samples/index.json

```diff
@@ -115,6 +115,12 @@
     "description": "Develop Node.js based applications in TypeScript. Includes Node.js, eslint, nvm, yarn, and the TypeScript compiler.",
     "gitUrl": "https://github.com/daytonaio/sample-typescript-node"
   },
+  {
+    "name": "React and Node.js Sample",
+    "description": "A sample project showcasing React and Node.js integration with Daytona.",
+    "repository": "https://github.com/Jitankasarkar/Sample-Template"
```

Comment:
Should be "gitUrl" instead of "repository"

Reply:
@Tpuljak Thank you for the feedback! I've updated the `index.json` file as per your suggestion (changed `repository` to `gitUrl`) and added a signed-off-by line to my commit. Please review the changes. Let me know if there's anything else I need to adjust!